### DISCUSSION TITLE
Integrating JS SDK inside GrapesJs.

### DISCUSSION THREAD

**Discussion by lifeofsa** [GrapesJS/grapesjs#5380] [Q&A] [answered]
Hi, I want to integrate a custom JavaScript SDK inside my GrapesJS application, basically my idea is to send the lead information of a form created with GrapesJS through SDK

**Comment by artf** [comment, marked as answer]
You can check [Components & JS](https://grapesjs.com/docs/modules/Components-js.html) guide on how to create components with JS
